app.py

```diff
@@ -2,41 +2,110 @@
 from flask_restful import Api, Resource
 from flasgger import Swagger
 import requests
-import prompt_processor
+from prompt_processor import PromptProcessor
+from data_handler import DataHandler
+
 app = Flask(__name__)
 app.config['SERVER_NAME'] = 'localhost:5000'
 swagger = Swagger(app)
 api = Api(app)
+
+# Create an instance of the DataHandler class
+data_handler = DataHandler('https://gptsnackdandies-default-rtdb.firebaseio.com/', 'gptsnackdandies-firebase-adminsdk-7ctws-a01d4f1f29.json')
+
 class DoctorCategoryResource(Resource):
     def post(self):
+        """
+        Process JSON input for doctor category
+        ---
+        parameters:
+          - name: body
+            in: body
+            required: true
+            schema:
+              type: object
+              properties:
+                payload:
+                  type: object
+                  properties:
+                    conversation:
+                      type: object
+                      properties:
+                        answers:
+                          type: object
+                          properties:
+                            answers:
+                              type: object
+                              properties:
+                                symptoms:
+                                  type: string
```

Comment:
do we need this structure here as part of the code? If not, please remove it and add it to a sample.txt file.

Reply:
yes we need this , based on his only - end point will allow the json fields

Reply:
can this be moved into another file instead of here and then pulled in when needed?